x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ssl/SSLServiceTests.java

```diff
@@ -144,7 +154,12 @@ public void testThatKeyStoreAndKeyCanHaveDifferentPasswords() throws Exception {
                 .put("xpack.ssl.keystore.path", differentPasswordsStore)
                 .setSecureSettings(secureSettings)
                 .build();
-        new SSLService(settings, env).createSSLEngine(Settings.EMPTY, Settings.EMPTY);
+        final SSLService sslService = new SSLService(settings, env);
+        SSLConfiguration configuration = sslService.sslConfiguration(Settings.EMPTY, Settings.EMPTY);
+        sslService.createSSLEngine(configuration, null, -1);
+
+        configuration = sslService.getSSLConfiguration("_global");
```

Comment:
Isn't the second `createSSLEngine()` redundant since `getSSLConfiguration("_global");` will return the same configuration as `sslConfiguration(Settings.EMPTY, Settings.EMPTY);` 
Possibly use `globalConfiguration()` as below ?

Reply:
Yes, this was one of the first tests that I converted to using named contexts, and then I realised that a random choice was cleaner than testing every possible option in each test.